user_manual/source/diffusion.ipynb

```diff
@@ -475,7 +475,7 @@
     "    tidx = mesh.findTetByPoint([xcrd, ycrd, zcrd])\n",
```

Comment:
<p>This change is not necessary. Can you please remove it from the PR?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CNS-OIST/STEPS_Example/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>